projects/extension/src/background/index.ts

```diff
@@ -183,6 +185,83 @@ chrome.runtime.onMessage.addListener(
   },
 )
 
+const updateDatabases = async () => {
+  const wellKnownChains = await loadWellKnownChains()
+  const client = smoldotStart({
+    cpuRateLimit: 0.5, // Politely limit the CPU usage of the smoldot background worker.
+  })
+
+  let dbChainsCounter = 0
+
+  let promises = []
+
+  for (const [key, value] of wellKnownChains) {
+    let databaseContent = await environment.get({
+      type: "database",
+      chainName: key,
+    })
+
+    const promise = new Promise<void>((resolve) => {
+      client
+        .addChain({
+          chainSpec: value,
+          databaseContent,
+        })
+        .then(async (chain) => {
+          chain.sendJsonRpc(
+            `{"jsonrpc":"2.0","id":"1","method":"chainHead_unstable_follow","params":[true]}`,
+          )
+
+          while (true) {
+            const response = JSON.parse(await chain.nextJsonRpcResponse())
+            if (response?.params?.result?.event === "initialized") {
+              chain.sendJsonRpc(
+                JSON.stringify({
+                  jsonrpc: "2.0",
+                  id: "2",
+                  method: "chainHead_unstable_finalizedDatabase",
+                  params: {
+                    max_size_bytes:
+                      chrome.storage.local.QUOTA_BYTES / wellKnownChains.size,
+                  },
+                }),
+              )
+            }
+            if (response?.id === "2") {
+              await environment.set(
+                { type: "database", chainName: key },
+                response.result,
+              )
+              dbChainsCounter++
+              resolve()
+              break
+            }
+          }
+        })
+    })
+    promises.push(promise)
+  }
+
+  Promise.all(promises).then(() => {
+    // Once the database content is saved in the localStorage for all the chains
+    // then terminate the client
+    if (dbChainsCounter === wellKnownChains.size) {
+      console.log("All databases are updated. Light Client is terminated.")
+      client.terminate()
+    }
+  })
+}
+
+updateDatabases()
```

Comment:
```suggestion
updateDatabases().catch(...)
```

If an error happens, we should mention that it happened while updating the databases.